### ISSUE TITLE
Add option to toggle swipe menu

### ISSUE THREAD

**Issue by yqu212** [orgzly/orgzly-android#730]
- [x] I have searched for existing issues that may be the same as or related to mine.

The swipe menu may cause misoperation in my daily use. The buttons do not give notifications. 

It's reasonable to add an option to toggle it since many users need it.

**Comment by ScottFreeCode** [comment]
I just hit the phone's back button when I've moved to a different view through the swipe menu, does that help in your case?

**Comment by yqu212** [comment]
No. The misoperation I met is that the buttons change the state of a entry without notification.
